openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -276,3 +276,39 @@ def get_queryset(self):
             )
 
         return invitations
+
+
+class UpdatePermissions(ListView):
+    """
+    UpdatePermissions View
+    """
+    def get(self, request, *args, **kwargs):
```

Comment:
According to internal guideline of the team, we leave a blank line after class doctsing